config/nwerc-2022/events.properties

```diff
@@ -0,0 +1,5 @@
+login=$creds.login
+password=$creds.password
+url=https://cds.chipcie.ch.tudelft.nl/api/contests/nwerc2022
```

Comment:
Do you have a cleared version of this file dump?

Reply:
Not yet, will add them when I have them

Reply:
*kind reminder*
Still not?